app/controllers/foreman_deployments/api/v2/stacks_controller.rb

```diff
@@ -20,6 +20,18 @@ def create
           process_response @stack.save
         end
 
+        api :PUT, '/stacks/:id/', N_('Update imported stack')
+        param_group :stack, :as => :update
+        param :id, :identifier, :required => true
+        def update
+          if @stack.configurations.empty?
```

Comment:
That's true. Currently config can be created only with a deployment. So probably we should enable deleting deployments for now. Deleting configs will be implemented when we add the "stored configs" feature. Does it make sense?